UI/EnvironmentGUIToggleLabel.cs

```diff
@@ -65,8 +67,11 @@ public class EnvironmentGUIToggleLabel : UIToggleImage {
 			else {
 				Text.TextColor = Color.Gray;
 				
-				string text = module?.DisabledTooltip.Value;
-				
+				//--1.4.4 TESTING--//
+				//Fix this to have proper localization
+				//string text = module?.DisabledTooltip.GetTranslation(Language.ActiveCulture);
+				var text = "DERP 1.4.4";
```

Comment:
need to replace with actual text retrieval method.  leaving this comment here as a reminder